api/models.py

```diff
@@ -229,10 +229,13 @@ class Country(models.Model):
         null=True, blank=True,
         max_length=255
     )
-    email = models.CharField(
-        verbose_name=_('Email'),
-        null=True, blank=True,
-        max_length=255
+    email = ArrayField(
```

Comment:
Noticed this late, but can we also change the email to emails? And use [] as default